subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/KotlinBuildScript.kt

```diff
@@ -49,8 +48,8 @@ import kotlin.script.templates.ScriptTemplateDefinition
         "-Xallow-unstable-dependencies",
         "-XXLanguage:+DisableCompatibilityModeForNewInference",
         "-XXLanguage:-TypeEnhancementImprovementsInStrictMode",
-    ],
-    provider = KotlinBuildScriptTemplateAdditionalCompilerArgumentsProvider::class
+        "-P=plugin:org.jetbrains.kotlin.assignment:annotation=org.gradle.api.SupportsKotlinAssignmentOverloading",
+    ]
```

Comment:
🤔 When changing script templates we should manually test the IDE integration again to make sure we don't break anything.

Reply:
Will do

Reply:
Checked locally and it seems it works ok. Also `=` is not marked as incubating anymore.